src/PromiseRouter.js

```diff
@@ -152,17 +155,30 @@ export default class PromiseRouter {
 // handler.
 // Express handlers should never throw; if a promise handler throws we
 // just treat it like it resolved to an error.
-function makeExpressHandler(promiseHandler) {
+function makeExpressHandler(appId, promiseHandler) {
+  let config = AppCache.get(appId);
   return function(req, res, next) {
     try {
-      log.verbose(req.method, maskSensitiveUrl(req), req.headers,
-                  JSON.stringify(maskSensitiveBody(req), null, 2));
+      let url = maskSensitiveUrl(req);
+      let body = maskSensitiveBody(req);
+      let stringifiedBody = JSON.stringify(body, null, 2);
+      log.verbose(`REQUEST for [${req.method}] ${url}: ${stringifiedBody}`, {
```

Comment:
Do we really want to stringify the entire body of the request and masking it, if the logging is not in verbose, can we guard it inside a ``if (VERBOSE)`` ?  

It seems like this would add a fair bit of time for big requests/responses.